client/webserver/site/src/css/forms.scss

```diff
@@ -19,4 +19,17 @@ table.marketstable {
     width: 33.3%;
     padding: 3px 0;
   }
+
+  tr.clickable {
+    cursor: pointer;
+    opacity: 0.5;
+  }
+
+  tr.clickable:hover {
+    opacity: 1;
+  }
+
+  tr.clickable.selected {
+    opacity: 1;
+  }
```

Comment:
I got the button padding and the selected row border with the following, but you can do it any way you like:

```suggestion
  button {
    padding: 4px 12px;
  }

  tr.clickable {
    cursor: pointer;
    opacity: 0.5;
  }

  tr.clickable:hover {
    opacity: 1;
  }

  tr.clickable.selected {
    opacity: 1;
    border: 1px solid #2d528c;
    border-radius: 1px;
  }
```

Reply:
I like this idea of the border, but I don't really like this blue. 

Tried something more gray, with #686767. what do you think?

Also removed border-radius, as it is not really doing anything

Reply:
Sure, gray sounds good.  I'll check it out in both dark and light modes to be sure, but I don't have a strong pref.

I noticed the radius wasn't working for me either but didn't look into it.  A little softer boundary would be nice, but it's fine.